src/Picqer/Financials/Exact/Connection.php

```diff
@@ -464,25 +464,25 @@ private function acquireAccessToken()
         }
     }
 
-    private function getDateTimeFromExpires($expires)
+    private function getTimestampFromExpiresIn($expiresIn)
     {
-        if (!is_numeric($expires)) {
+        if (!is_numeric($expiresIn)) {
```

Comment:
Unrelated to your PR, but it may be feasible to use `ctype_digit()` here rather than `is_numeric()`. According to https://tools.ietf.org/html/rfc6749#section-5.1 , this field contains the number of seconds so `is_numeric()` may allow a bit more than we would want.

Reply:
Again, I actually just wanted to remove this function.

I don't think Exact would ever return anything other than an int. And if they do, it's a horrible bug on their side which you shouldn't attempt to handle IMO.

Reply:
Although on a lot of other places we do rely on what Exact returns as documented, I think in this case `ctype_digit` is a better check. Could you please change the check? I'd love to have that improvement within this PR.

Reply:
Well, since the `+` operator is used on it, anything invalid and potentially dangerous would still just yield an int or a float and therefore be pretty much harmless, but I digress.